src/main/java/net/onelitefeather/antiredstoneclockremastered/service/RedstoneClockService.java

```diff
@@ -169,7 +171,9 @@ private void sendNotification(final Player player, final Location location) {
     private void breakBlock(@NotNull Location location) {
         Block block = location.getBlock();
         if (this.dropItems) {
-            var drops = block.getDrops();
+            ItemStack destroyTool = new ItemStack(Material.DIAMOND_PICKAXE);
```

Comment:
Don't re-create for each block again a new item stack. That call has the possibilities to create X new Objects in the Java Memory.

Reply:
New commit added